schemas/register-entry-schema.json

```diff
@@ -0,0 +1,113 @@
+  "properties": {
+    "subject_entity": {
+      "$ref": "includes/entity.json"
+    },
+    "register": {"$ref": "#/definitions/register"},
+    "identifier": {
+      "description": "A unique identifier given to the register entry by the register. In some cases – e.g. Charity Register for England & Wales, these identifiers (in this case known as the 'charity number') are used outside of the context of the register",
+      "type": "string"
+    },
+    "start_date": {
+      "description": "The date the register entry started, e.g. the date the entity was registered from, or appeared on the register the first time",
+      "type": "string",
+      "format": "date"
+    },
+    "end_date": {
+      "description": "The date the register entry ended, e.g. the date the charity was deregistered from",
+      "type": "string",
+      "format": "date"
+    },
+    "source_url": {
+      "description": "Source of the data – this may be the url of the entity on the register, the URL of an open data dump, or if there is no permanent URLs, the main URL for the register",
+      "type": "string",
+      "minLength": 8
+    },
+    "register_url": {
+      "description": "URL of the entity on the register, if there is a permanent URL",
+      "type": "string",
+      "minLength": 8
+    },
+    "confidence": {
+      "description": "Confidence in accuracy of data",
+      "type": "string",
+      "enum": [
+        "HIGH",
+        "MEDIUM",
+        "LOW"
+      ]
+    },
+    "sample_date": {
+      "description": "Date on which we know this to be true (usually date this information was retrieved from the source)",
+      "type": "string",
+      "format": "date"
+    },
+    "retrieved_at": {
+      "description": "The time or date at which the source URL was requested",
+      "type": "string",
+      "format": "date"
+    },
+    "status": {
+      "type": "string",
+      "description": "The status of the register entry"
+    },
+    "category": {
+      "description": "Category of register entry (NB some registers may have more than one type of entry, so we record at the register entry level, rather than the register level)",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": [
+          "Charity-NonProfit",
```

Comment:
@sebbacon I think probably we should require this to be a single category rather than an array, as we want every entry to have one (and only one?) category. Can't think of example that would have more than one.